js/example/VCData.js

```diff
@@ -0,0 +1,45 @@
+const VC = {
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2",
+    "https://w3id.org/vc/render-method/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
```

Comment:
credential type says it's university degree but credentialSubject says it's insurance credential.
Have realistic example

Reply:
Updated with Insurance sample data.